sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.institutions.dto.ts

```diff
@@ -8,3 +10,28 @@ export class ApplicationOfferingChangeSummaryAPIOutDTO {
   studyEndPeriod: string;
   fullName: string;
 }
+
+/**
+ * Inprogress application offering change details.
+ */
+export class InprogressApplicationOfferingChangesAPIOutDTO {
+  applicationNumber: string;
+  applicationId: number;
+  studyStartPeriod: string;
```

Comment:
the property name ending with period could be changed to date. `studyStartDate` ?

same for `studyEndPeriod`?